hive-plugin/src/main/java/com/wipro/ats/bdre/hiveplugin/hook/LineageHook.java

```diff
@@ -82,7 +73,7 @@ public void run(HookContext hookContext) {
 
 	    } catch (Throwable e) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Catch Exception instead of Throwable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1181)